### ISSUE TITLE
Does not work in WSL

### ISSUE THREAD

**Issue by KoryNunn** [juliangruber/tape-run#79]
Hey, I realise I can't provide much info but when I try tape-run on [WSL](https://docs.microsoft.com/en-us/windows/wsl/faq) it fails without any output. The identical setup works on ubuntu, and in travis ci.

Example of a repo that works everywhere but WSL: https://github.com/KoryNunn/scroll-into-view/

Example output from WSL: 
```
/scroll-into-view$ npm test

> scroll-into-view@1.12.2 test /scroll-into-view
> browserify ./test/test.js | tape-run

npm ERR! Test failed.  See above for more details.
```

If I had to guess it'd be because wsl runs fully headless, and there might be some assumptions being made by electron (or it's configuration) around display output.

**Comment by juliangruber** [comment]
I currently don't have access to a windows machine (my GPU is being repaired), so I'll try to guess debug this.

Could you try this suggestion? https://github.com/electron-userland/electron-prebuilt/issues/260

Also there's https://github.com/GoogleChrome/puppeteer/issues/1837

**Comment by fregante** [comment]
I think this has to be fixed upstream in https://github.com/substack/browser-launcher

Do you mind testing that package an opening an issue there? The package explicitly supports Windows but there's no mention of WSL

**Comment by KoryNunn** [comment]
Yep it doesn't detect a browser in WSL, i'll put an issue up there.
